azuredeploy.json

```diff
@@ -0,0 +1,403 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "functionAppName": {
+            "type": "string",
+            "metadata": {
+                "description": "Function App Nam. Must be globally unique."
+            },                        
+            "maxLength": 24
+        },
+        "storageAccountName": { 
+            "type": "string",
+            "metadata": {
+                "description": "Storage Account Name (lowercase). Must be globally unique"
+            },         
+            "minLength": 0,
+            "maxLength": 9
+        },
+        "hostingPlanName": {
+            "type": "string",
+            "metadata": {
+                "description": "Hosting Service Name. Globally unique."
+            },            
+            "minLength": 3,
+            "maxLength": 24
+        },
+        "blobContainerPrefix": {
+            "type":"string",
+            "metadata": {
+                "description": "the prefix that will be used in the naming of containers in Blob storage for each region your app/server supports. The default is 'region'"
```

Comment:
```suggestion
                "description": "the prefix that will be used in the naming of containers in Blob storage for each region your app/server supports. The default is 'region-'"
```

Reply:
Kinda the same comment as before... the hyphen will get appended on during the deployment process, so we don't have to rely on anybody to remember to enter it.